types/mochawesome/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for mochawesome 6.2
+// Project: https://github.com/adamgruber/mochawesome#readme
+// Definitions by: Chris Gilardi <https://github.com/Christop406>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface ReporterOptions {
+    quiet: boolean;
+    code: boolean;
+    'no-code': boolean;
+    html: boolean;
+    json: boolean;
+    consoleReporter: string;
+}
+
+interface ConfigOptions {
+    inlineDiffs?: boolean;
+    reporterOptions?: Partial<ReporterOptions>;
+}
+
+/**
+ * Initialize a new reporter.
+ *
+ * @api public
+ */
+declare function Mochawesome(runner: Mocha.Runner, options?: ConfigOptions): void;
```

Comment:
I'm pretty sure that this function is actually a constructor as it relies heavily on `this`. It should only be callable with a `new` keyword. Unless I misunderstand something, I don't really know how mocha works.

Reply:
Hey sorry, didn't see these comments. Didn't mean to update yet. Will test this out and make the change or let you know if that's not the case.